backend/coins/btc/account.go

```diff
@@ -99,13 +99,15 @@ type Account struct {
 	activeTxProposal     *maketx.TxProposal
 	activeTxProposalLock locker.Locker
 
-	feeTargets []*FeeTarget
+	feeTargets   []*FeeTarget
+	feeTargetsMu locker.Locker
```

Comment:
I see that in this structure we use `Locker` or `Mutex` in different places. Would be possible to uniform it and use Lockers everywhere? Apart from this: `feeTargetsMu` suggests that it is a Mutex. Maybe change to `feeTargetsLock` instead? Same for `initializedMu`

Reply:
Done